update README (#1198)
ocaml/merlin: README.md

### Other editors

> Merlin only supports Vim and Emacs out-of-the-box. Below is a short reference of getting

Reviewer: ```suggestion
Merlin only supports Vim and Emacs out-of-the-box. This section describes shortly how to get
```

Reviewer: It sounds a tad better IMO

merlin-based editor support in other editors.

#### Visual Studio Code

OCaml has official support for Visual Studio Code extension called `OCaml Platform` available in the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=ocamllabs.ocaml-platform). Project source is available [here](https://github.com/ocamllabs/vscode-ocaml-platform).
*Note* that it requires installed [OCaml-LSP](https://github.com/ocaml/ocaml-lsp), an official
[Language Server Protocol(LSP)](https://microsoft.github.io/language-server-protocol/specifications/specification-current/)
implementation for OCaml based on merlin.

> #### Rest

Reviewer: ```suggestion
#### Editors without official support
```
